docs-src/server/serve-docs.ts

```diff
@@ -11,6 +11,7 @@ async function rebuildDocs() {
 	console.log('\n🔄 Rebuilding docs...')
```

Comment:
@fabianhaef I think we should go for a more fine-grained approach to rebuilding docs files here. We know:

- `*.ts` file in `docs-src/components/` changed -> `build:docs-js`
- `*.css` file in `docs-src/components/` changed -> `build:docs-css`
- `*.html` file in `docs-src/components/` changed -> `build:docs-html`
- `*.md` file in `docs-src/pages/` changed -> `build:docs-html`
- `*.ts` file in `src/` changed -> `build && build:docs-js && build:docs-api`